app/src/main/java/com/eveningoutpost/dexdrip/Models/LibreBlock.java

```diff
@@ -48,15 +51,15 @@
     public byte[] blockbytes;
 
     // if you are indexing by block then just * 8 to get byte start
-    public static LibreBlock createAndSave(String reference, byte[] blocks, int byte_start) {
```

Comment:
Really this signature method should remain and call the updated one with the current timestamp to preserve the original behavior and avoid having to refactor the callers.

Reply:
The problem with the current signature, is that we have a libreblock, and a bg-record that should have the same timestamp.
If each function will call JoH.tsl() by itself, it is not guaranteed that they will have the same value.
This is why I prefer to take the extra effort and change the signature of the function. (timestamp is used as the unique key between the two tables).